test/test_encounters.py

```diff
@@ -0,0 +1,135 @@
+from datetime import datetime, timedelta, timezone
+from xml.etree.ElementTree import Comment
+
+import pytest
+
+from welkin.exceptions import WelkinHTTPError
+from welkin.models.encounter import Encounter, Encounters
+from welkin.util import clean_datetime
+
+UTC = timezone.utc
+
+
+@pytest.mark.vcr()
+def test_encounter_create(client, vcr_cassette):
+    patient = client.Patient(id="0056c34b-9a83-41bd-bf4d-e4710d7a77f9")
+    start = datetime.now(tz=UTC) + timedelta(hours=6)
+    end = start + timedelta(hours=1)
+
+    data = {
+        "title": "new enc",
+        "description": "new encounter",
+        "templateName": "etmp-encounter1",
+        "currentScheduledAppointment": {
+            "eventType": "ENCOUNTER",
+            "eventTitle": "NEW ENC",
+            "startDateTime": clean_datetime(start),
```

Comment:
Don't use the `clean_datetime` directly, it's called as part of `Client.prepare_request`